keyboards/aya/readme.md

```diff
@@ -0,0 +1,15 @@
+# Aya
+
+![Aya](https://raw.githubusercontent.com/DanNixon/aya/main/docs/both_sides.jpg)
+
+A column staggered, symmetrical split keyboard.
+
+* Keyboard Maintainer: [Dan Nixon](https://github.com/dannixon)
+* Hardware Supported: Aya PCB, Pro Micro
+* Hardware Availability: CAD files [here](https://github.com/DanNixon/aya)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```